pcsx2/gui/AppMain.cpp

```diff
@@ -1038,6 +1038,12 @@ void Pcsx2App::CloseGsPanel()
 		if (GSPanel* woot = gsFrame->GetViewport())
 			woot->Destroy();
 	}
+#ifndef DISABLE_RECORDING
+	// Disable recording controls that only make sense if the game is running
+	sMainFrame.enableRecordingMenuItem(MenuId_Recording_FrameAdvance, false);
+	sMainFrame.enableRecordingMenuItem(MenuId_Recording_TogglePause, false);
+	sMainFrame.enableRecordingMenuItem(MenuId_Recording_ToggleRecordingMode, false);
+#endif
```

Comment:
Sorry, I don't understand the move from `OnGsFrameClosed` to here, what does this accomplish?  In my testing, the menu options are disabled when the GS window is closed.

Reply:
I see your explanation on the parent PR.  I'll take a look, can probably ignore my comments.